test/e2e/app/app.go

```diff
@@ -462,7 +462,7 @@ func (app *Application) ExtendVote(_ context.Context, req *abci.RequestExtendVot
 		ext = make([]byte, 8)
 		if num, err := rand.Int(rand.Reader, big.NewInt(voteExtensionMaxVal)); err != nil {
 			panic(fmt.Errorf("could not extend vote. Len:%d", len(ext)))
-		} else {
+		} else { //nolint: revive
```

Comment:
Can we just move this one out of the else?

Reply:
The problem is that `num` scope is the if-else, so if we remove the else, then we have to define num outside the `if`. I think the way it was done was more idiomatic (leveraging go scopes)

Reply:
I actually prefer the `if-else` but am trying to get more go idiomatic :)
But I am ok with either way here.